### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by dmitry** [felipecsl/wombat#27]
Wombat have very good DSL, but lacks some stuff for crawling, like going through the pages. Do you have any roadmap or plans, on which parts of the wombat you will be working next?

**Comment by fabn** [comment]
+1 for this

**Comment by dmitry** [comment]
Someone else are also requested this feature here: https://github.com/felipecsl/wombat/issues/34

**Comment by felipecsl** [comment]
I agree this gem could use some love (eg. more features). #34 is definitely on the radar. As usual, I'm open for contributions if you're up for it, otherwise, I'll tackle this as soon as I have some time.

**Comment by justi** [comment]
+1

**Comment by hgani** [comment]
+1
